src/OpenSSL/crypto.py

```diff
@@ -1518,6 +1518,18 @@ def set_flags(self, flags):
         """
         _openssl_assert(_lib.X509_STORE_set_flags(self._store, flags) != 0)
 
+    def set_time(self, vfy_time):
+        """Set verification time to this store.
+
+        .. versionadded: 16.3.0
+
+        :param datetime vfy_time: The verification time to set on this store.
+        :return: ``None`` if the verification time were successfully set.
+        """
+        param = _lib.X509_VERIFY_PARAM_new()
```

Comment:
Thanks for your feedback!

When I try:
```python
from OpenSSL._util import lib
lib.X509_VERIFY_PARAM_free
```
I got this exception on my system:  `AttributeError: 'module' object has no attribute 'X509_VERIFY_PARAM_free'`

It looks like `X509_VERIFY_PARAM_free` is not exported, I should do a PR on cryptography to export it? or something is wrong on my side?

Thanks!